### PR TITLE
Db/sync extremes

### PR DESCRIPTION
**Pull request by goransle**
Removed separate panning sync option/handler, renamed `selection` to `extremes`. 

This extremes sync has been reworked to implement DataCursor. Note that Highcharts <-> Highcharts sync uses setExtremes event data, while Highcharts -> DataGrid uses the dataCursor row.

### CODE REVIEW

File: samples/cypress/dashboards/sync/chart-extremes/demo.details [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: test
+js_wrap: b
+authors:
+  - Torstein Hønsi
```
Comment (karolkolodziej) [inline samples/cypress/dashboards/sync/chart-extremes/demo.details:5]: Are you sure?

**Review by karolkolodziej — CHANGES_REQUESTED** [review]
In `dashboards/demos/axes-sync` I'm getting:
image

**Comment by bre1470** [comment]
In master / master merge it became `Dashboards.DataConnector.registry.CSV`, so maybe recompile to fix the issue?

**Comment by bre1470** [comment]
Reset Zoom is not working for me in axes-sync.

**Comment by goransle** [comment]
> 

As in it does not reset the chart you are clicking the button on? Or not resetting the linked chart? The latter is intended to simplify things, but I can probably bring back the reset-sync.

**Review by pawellysy — APPROVED** [review]

**Review by karolkolodziej — CHANGES_REQUESTED** [review]
I'm also in favour of syncing the Reset zoom button.

**Review by bre1470 — CHANGES_REQUESTED** [review]
This PR would restore regressions

**Comment by goransle** [comment]
Visual comparison error is likely due to outdated `highcharts-utils` on `dashboard/main`

**Review by sebastianbochan — APPROVED** [review]

**Pull request merged**